src/ape/pytest/plugin.py

```diff
@@ -10,6 +10,16 @@
 from ape.utils.basemodel import ManagerAccessMixin
 
 
+def _get_default_network() -> str:
+    default_ecosystem = ManagerAccessMixin.network_manager.default_ecosystem
+    if default_ecosystem.default_network.is_mainnet:
+        # Don't use mainnet for tests, even if it configured as
+        # the default.
+        return "ethereum:local:test"
```

Comment:
I think if this is true, we should just raise to tell the user of a configuration error instead of proceeding